### ISSUE TITLE
Correctly handle header field "Accept"

### ISSUE THREAD

**Issue by adius** [chubin/wttr.in#145]
If I send a request with `Accept: text/plain` I would expect it to return the plain text version.
Instead I have to set `User-Agent: curl/7.54.0`, which is very hacky.

It should be possible to overwrite the default format with the `Accept` header field.

**Comment by chubin** [comment]
Nice idea! What about terminal sequences? Should they be on or off by default if you do "Accept: text/plain"? (ansonsten ist die Idee echt gut)

**Comment by adius** [comment]
Uhm, very good question. I guess they should actually be off.
… but what would be the correct header field to activate them?
(Immer diese Deutschen 😂)

**Comment by chubin** [comment]
Maybe text/x-ansi?

**Comment by adius** [comment]
Yeah, sounds like a good idea! 😊

**Comment by adius** [comment]
I just saw that the `x-` is deprecated: https://www.rfc-editor.org/bcp/bcp178.txt

So let's better use `text/ansi` I guess.

**Comment by chubin** [comment]
Indeed. Ok, so let it be `text/ansi`

**Comment by Tachi107** [comment]
> I just saw that the `x-` is deprecated: https://www.rfc-editor.org/bcp/bcp178.txt
> 
> So let's better use `text/ansi` I guess.

Yeah, using `x-` for new types is deprecated, but using `text/ansi` when that media type doesn't exist (it's not registered with IANA) is even worse. You could use `text/x.ansi`, or `text/vnd.wttr.ansi`, even though "ansi" is a bit generic, so I'd suggest something like `text/vnd.wttr.term`, or `vnd.wttr.vt` (for "video terminal", like the original VT100 and similar)

**Comment by chubin** [comment]
Thank you for raising this.

Also, here's a relevant discussion that might be helpful:

https://news.ycombinator.com/item?id=44590189
